Cargo.toml

```diff
@@ -19,6 +19,7 @@ pyo3 = { version = "0.21", features = [
     "abi3-py39",
 ] }
 num-integer = "0.1"
+jax = "0.6.2"
```

Comment:
`Cargo.toml` is for Rust dependencies. You need to add `jax` to `pyproject.toml` instead.